docs/modelserving/v1beta1/custom/custom_model/README.md

```diff
@@ -75,6 +75,25 @@ class AlexNetModel(kserve.Model):
 if __name__ == "__main__":
     kserve.ModelServer().start({"custom-model": AlexNetModel})
 ```
+gpu example
+```python
+@serve.deployment(name="custom-model", num_replicas=2, ray_actor_options={"num_cpus":1, "num_gpus": 0.5})
```

Comment:
Does Ray support fractional gpu ?

Reply:
@yuzisun 
Yes, ray supports fractional gpu.
https://docs.ray.io/en/latest/serve/scaling-and-resource-allocation.html#serve-omp-num-threads
![image](https://user-images.githubusercontent.com/28818972/188877612-208404ac-3e3e-4563-9bfa-e4ab2f27bee0.png)

Reply:
@park12sj Sorry for the late reply, can you help add this reference so people can read more details from Ray website?